docs/sql-reference/sql-statements/data-definition/CREATE TABLE.md

```diff
@@ -270,30 +270,51 @@ For more information, see [Data distribution](../../../table_design/Data_distrib
+
+  - If the query is complex, we recommend that you select a high cardinality column as the bucketing column to ensure balanced data distribution among buckets and improve cluster resource utilization.
+  - If the query is relatively simple, we recommend that you select the column that is often used as the query condition as the bucketing column to improve query efficiency.
+
+  If partition data cannot be evenly distributed into each bucket by using one bucketing column, you can choose multiple bucketing columns (at most three). For more information, see [Choose bucketing columns](../../../table_design/Data_distribution.md).
+
+  **Precautions**:
 
-Data in partitions can be subdivided into tablets based on the hash values of the bucketing columns and the number of buckets. We recommend that you choose the column that meets the following two requirements as the bucketing column.
+  - **When you create a table, you must specify the bucketing columns**.
+  - The values of bucketing columns cannot be updated.
+  - Bucketing columns cannot be modified after they are specified.
+  - Since StarRocks 2.5.7, you do not need to set the number of buckets when you create a table. StarRocks automatically sets the number of buckets. If you want to set this parameter, see [Determine the number of buckets](../../../table_design/Data_distribution.md#determine-the-number-of-buckets).
 
-- High cardinality column such as ID
-- Column that is often used as a filter in queries
+- Random bucketing (since v3.1)
 
-If such a column does not exist, you can determine the bucketing column according to the complexity of queries.
+  For data in a partition, StarRocks distributes the data randomly across all buckets, which is not based on specific column values. And if you want the StarRocks to automatically determine the number of buckets, you do not need to specify any bucketing information. If you choose to manually specify the number of buckets, the syntax is as follows:
 
-- If the query is complex, we recommend that you select a high cardinality column as the bucketing column to ensure balanced data distribution among buckets and improve cluster resource utilization.
-- If the query is relatively simple, we recommend that you select the column that is often used as the query condition as the bucketing column to improve query efficiency.
+  ```SQL
+  DISTRIBUTED BY RANDOM BUCKETS <num>
+  ```
 
-If partition data cannot be evenly distributed into each tablet by using one bucketing column, you can choose multiple bucketing columns (at most three). For more information, see [Choose bucketing columns](../../../table_design/Data_distribution.md).
+  However, note that random bucketing may not be suitable for scenarios that involve querying and aggregating based on specific columns. In such cases, hash bucketing may be more appropriate as it can store similar data in the same bucket, facilitating data access and processing.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/3cce73e5-9dbd-42d9-ab99-5a84bf42ae25)
-> queries and aggregations